man/coap-server.txt.in

```diff
@@ -14,11 +14,12 @@ coap-server - CoAP Server based on libcoap
 
 SYNOPSIS
 --------
-*coap-server* [*-A* addr] [*-g* group] [*-p* port] [*-v* num]
+*coap-server* [*-A* addr] [*-g* group] [*-p* port] [*-l* loss]
+              [*-k* key] [*-c* certfile] [*-v* num]
 
 DESCRIPTION
 -----------
-*coap-server* is a CoAP server which simulate 6LoWPAN devices which can be
+*coap-server* is a CoAP server which simulates 6LoWPAN devices which can be
```

Comment:
As the constrained devices space is not defined by 6LoWPAN anymore, we should probably rephrase that sentence. How about:

"*coap-server* is an example server for the 'Constrained Application Protocol` (RFC 7252)."

Reply:
Updated